pkg/config/config.go

```diff
@@ -42,6 +42,8 @@ type Feed struct {
 	Clean Cleanup `toml:"clean"`
 	// Custom is a list of feed customizations
 	Custom Custom `toml:"custom"`
+	// Included in OPML file
+	OPML bool `toml:"opml"`
```

Comment:
Would it make sense to have this switch globally instead of per-feed?

Reply:
It make perfect sense however per-feed gives more granularity with little effort.